client/src/Picks.tsx

```diff
@@ -1026,26 +1071,44 @@ function Description({
       .then((response) => onNominationUpdate(response.body))
       .then(then)
       .catch(alertApiErrorMessage)
-      .finally(() => setEditing(false));
+      .finally(() => {
+        setEditing(false);
+        setShowHistory(false);
+      });
   };
 
   const editors = edits
     .map((edit) => edit.editor)
     .filter((u1, i, all) => u1.id !== author?.id && all.findIndex((u2) => u1.id === u2.id) === i);
 
   return editing ? (
-    <Form busyState={[busy, setBusy]} onSubmit={onSubmit}>
-      <div className='textarea-wrapper'>
-        <textarea name='description' defaultValue={text} ref={descriptionRef} />
-        <div className='description-buttons'>
-          <span>Use BBCode for formatting</span>
-          <button type='submit'>{busy ? 'Updating...' : 'Update'}</button>
-          <button type='button' onClick={() => setEditing(false)}>
-            Cancel
-          </button>
+    <>
+      <DescriptionHistory
+        author={author}
+        edits={edits}
+        showHistory={showHistory}
+        setShowHistory={setShowHistory}
+      />
+      <Form busyState={[busy, setBusy]} onSubmit={onSubmit}>
+        <div className='textarea-wrapper'>
+          <textarea name='description' defaultValue={text} ref={descriptionRef} />
+          <div className='description-buttons'>
+            <button
+              type='button'
+              disabled={edits.length === 0}
```

Comment:
also now that i think about it, maybe `edits.length <= 1` for condition ? history with one item is not very helpful

Reply:
i'm not sure at all why this happens and troubleshooting it hasn't really done much so weird

Reply:
forgot that `Form` does some stupid hack to enable/disable controls based on busy state so thats where bug comes from. will just let that be for now since im removing or replacing `Form` soon (#166)